perllib/FixMyStreet/App/Controller/Auth/Profile.pm

```diff
@@ -74,8 +74,7 @@ sub change_password : Path('/auth/change_password') {
 
     if ($c->user->password) {
         # we should have a usable password - save it to the user
-        $c->user->obj->set_password($new, 1);
-        $c->user->obj->update;
+        $c->user->obj->update( { password => $new } );
```

Comment:
This doesn't seem to set `last_password_change` because it's not calling `password`.